qiskit/pulse/schedule.py

```diff
@@ -711,13 +712,18 @@ def is_parameterized(self) -> bool:
         return self._parameter_manager.is_parameterized()
 
     def assign_parameters(
-        self, value_dict: dict[ParameterExpression, ParameterValueType], inplace: bool = True
+        self,
+        value_dict: dict[
+            ParameterExpression | ParameterVector, ParameterValueType | list[ParameterValueType]
+        ],
+        inplace: bool = True,
     ) -> "Schedule":
         """Assign the parameters in this schedule according to the input.
 
         Args:
-            value_dict: A mapping from Parameters to either numeric values or another
-                Parameter expression.
+            value_dict: A mapping from Parameters (ParameterVectors) to either
```

Comment:
It was already using "Parameters" but I think here I would use "parameters" and "parameter vectors" unless you want to use Sphinx references to make it like `` :class:`.Parameter` ``s